packages/components/src/primitives/svg/index.native.js

```diff
@@ -1,13 +1,26 @@
 import {
 	Path as PATH,
+	G as GROUP,
 	Svg,
 } from 'react-native-svg';
 
 export const Path = PATH;
+export const G = GROUP;
 export const SVG = ( props ) => {
-	return (
-		<Svg width={ props.width } height={ props.height } >
-			{ props.children }
-		</Svg>
-	);
+	if (props.width != undefined && props.height != undefined) {
+		return (
+			<Svg width={ props.width } height={ props.height } >
+				{ props.children }
+			</Svg>
+		);
+	} else {
```

Comment:
Svg implementation handles `viewBox` out of the box:
https://github.com/react-native-community/react-native-svg/blob/master/elements/Svg.js#L35

Reply:
That's right `viewBox` is handled, but the component will fail to render if `height`  and `width` are not set - what this  (most probably hack-ish) piece of code does is fabricate the height and width out of the information passed in `viewBox`  which seemed to exist in all cases for all blocks in code (I relied on that to process unless `width` and `height` are explicitely set through props). For this I separated this specific change in commit c75a0db13fe675b80cfd869aa926256b0a7c5553  wdyt?